src/test/java/seedu/address/testutil/PastAppointmentBuilder.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.testutil;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Set;
+
+import seedu.address.model.appointment.PastAppointment;
+import seedu.address.model.tag.Medication;
+import seedu.address.model.util.SampleDataUtil;
+
+/**
+ * A utility class to help with building PastAppointment objects.
+ */
+public class PastAppointmentBuilder {
+    public static final String DEFAULT_DATE = "06-12-2022";
+    public static final String[] DEFAULT_MEDICATION = { "Paracetamol", "Ibuprofen" };
+    public static final String DEFAULT_DIAGNOSIS = "Fever";
```

Comment:
I think these testcase fields are meant to be used only within this class, so it should be private.
I realised there were multiple occurrences of a String of date, diagnosis and medication that needs to be used for your testcases, therefore you might want to use these for other Appointment related test cases.

Reply:
Done! Kept it private to be usable in other classes.